### DISCUSSION TITLE
Where to buy a domain? از کجا دامنه بخریم؟

### DISCUSSION THREAD

**Discussion by meness** [hiddify/Hiddify-Manager#3591] [Q&A سوالات رایج]
سلام،

پیشنهاد میکنید دامنه IR بخاطر سهولت و قیمت بخریم و روی Cloudflare تنظیمش کنیم؟ یا دامنه COM رو پیشنهاد میکنید؟

**Comment by AliAshooriyoon** [comment]
اگر واست بحث قیمت مطرحه میتونی از سایت زیر با تقریبا  1 دلار انواع دامین هارو تهیه بکنی

https://www.namecheap.com/promos/99-cent-domain-names/
